llama_stack/apis/post_training/post_training.py

```diff
@@ -88,10 +88,8 @@ class QATFinetuningConfig(BaseModel):
     group_size: int
 
 
-AlgorithmConfig = register_schema(
-    Annotated[LoraFinetuningConfig | QATFinetuningConfig, Field(discriminator="type")],
-    name="AlgorithmConfig",
-)
+_AlgorithmConfig = Annotated[Union[LoraFinetuningConfig | QATFinetuningConfig], Field(discriminator="type")]
```

Comment:
You have Union of a Union going in there?

Reply:
Huh yeah, though it doesn't seem like typing system or API docs generator care. Probably a union of two types is equivalent to a union of a union of these two types.